components/search/src/lib.rs

```diff
@@ -55,34 +52,6 @@ fn path_tokenizer(text: &str) -> Vec<String> {
         .collect()
 }
 
-fn build_tokenizers(search_config: &Search, language: Language) -> Vec<TokenizerFn> {
-    let text_tokenizer = match language {
-        #[cfg(feature = "indexing-zh")]
-        Language::Chinese => pipeline::tokenize_chinese,
-        #[cfg(feature = "indexing-ja")]
-        Language::Japanese => pipeline::tokenize_japanese,
-        _ => pipeline::tokenize,
-    };
```

Comment:
Additional languages gated by features work automatically now?

Reply:
Tokenization is now included in the language trait.

Reply:
Looks good then, I'll merge it when 3.0 is released